comp/core/wmcatalog/def/collector.go

```diff
@@ -3,20 +3,21 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-package workloadmeta
+// Package wmcatalog defines the catalog of collectors for workloadmeta
+package wmcatalog
 
```

Comment:
Do you plan on adding a `Component` interface to this component?

Reply:
Yes, as mentioned in the PR description.